frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -45,13 +46,21 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values }) => {
           </ProgressiveListItem>
         )}
         {/* Hide Deployment for Serverless */}
-        {values.serverless.enabled ? null : (
-          <ProgressiveListItem name="Deployment Configuration">
+        {values.resources !== Resources.KnativeService && (
+          <ProgressiveListItem
+            name={
+              values.resources === Resources.Kubernetes ? 'Deployment' : 'Deployment Configuration'
```

Comment:
Oh no! This is my bad, I misunderstood UX, I guess I forgot to update my comment. Steve S and Serena agreed this should just be called `Deployment`. Terribly sorry.

Reply:
i've corrected the changes

Reply:
Thanks for the quick turn around, sorry again. I thought I had updated my comment.